packages/lodestar/src/api/impl/validator/validator.ts

```diff
@@ -75,6 +75,10 @@ export class ValidatorApi implements IValidatorApi {
     randaoReveal: Bytes96,
     graffiti = ""
   ): Promise<BeaconBlock> {
+    if (!this.sync.isSynced()) {
+      const syncStatus = this.config.types.SyncingStatus.toJson(await this.sync.getSyncStatus());
+      throw new Error(`Node is syncing, status: ${JSON.stringify(syncStatus)}`);
```

Comment:
Can you throw ApiError that contains prop statusCode and throw it here with status code set to 503.
Rest api should handle this error and return proper status code

Reply:
I switched to using our middlware, it used 503 there.
Since we use it in single place, I don't think we need ApiError?

Reply:
I don't think we should use middleware as we might consume this api directly (not trough rest)